sdk/go/common/workspace/paths.go

```diff
@@ -269,6 +295,24 @@ func isProject(path string) bool {
 	return isMarkupFile(path, ProjectFile)
 }
 
+// Given a directory path search for files that appear to be a valid project that is satisfy [isProject].
+func findPluginInDir(dir string) (string, bool) {
+	// Check all supported extensions.
```

Comment:
Do we support anything other than `.yaml` extentions for `PulumiPlugin` files?  E.g. in https://github.com/pulumi/pulumi/blob/master/sdk/go/common/workspace/plugins.go#L1926, we hardcode `PulumiPlugin.yaml`.  Should we fix that?  Or can we just not support other extensions?

Reply:
Yeh we should fix them to allow yml and json as well I suppose like Pulumi files